client/components/Editor/plugins/malformedDoc.ts

```diff
@@ -60,15 +60,8 @@ export default () => {
 				const transactionMetadata = getTransactionMetadata(tr, state);
 				if (!hasCaptured) {
 					hasCaptured = true;
-					let theError: Error;
-					try {
-						throw new Error('Uh-oh!');
-					} catch (err: any) {
-						theError = err;
-					}
-					Sentry.captureEvent({
-						message: 'Evil Prosemirror Transaction',
-						contexts: { transactionMetadata, error: theError },
+					Sentry.captureException(new Error('Evil Prosemirror Transaction'), {
+						contexts: { transactionMetadata },
```

Comment:
This refactor was to fix a type error. I think it's mostly equivalent but let me know if I misunderstood the goal of this block!

Reply:
No, thank you for removing this ridiculous little bit of code!

Reply:
I have re-read the previous implementation here a few times and can't reason why there was a try-catch block there. It reads like a code-koan.
"Try to fail, and if you do, you have your answer, so continue."